src/components/v2/V2Create/ProjectPreview.tsx

```diff
@@ -55,7 +56,10 @@ export default function ProjectPreview({
     fundingCycleMetadata,
 
     distributionLimit: fundAccessConstraint?.distributionLimit,
-    distributionLimitCurrency: fundAccessConstraint?.distributionLimitCurrency,
+    distributionLimitCurrency:
+      !fundAccessConstraint?.distributionLimitCurrency.eq(0)
```

Comment:
Added this here because the currency was coming through as 0 which was causing all the `DistributionRatio`s to render nothing